splinterd/tests/admin/mod.rs

```diff
@@ -14,6 +14,8 @@
 
 //! Admin service integration tests.
 
+#[cfg(feature = "node")]
```

Comment:
Could this features guard be moved up a level into admin_service.rs. Nothing in this file is not guarded by "node".

Reply:
Probably, I don't remember specific reasoning for guarding each individually

Reply:
I added a commit to move that feature guard to the top-level module import